shotgun_api3/shotgun.py

```diff
@@ -3575,7 +3546,7 @@ def _make_call(self, verb, path, body, headers):
             attempt += 1
             try:
                 return self._http_request(verb, path, body, req_headers)
-            except ssl_error_classes as e:
+            except (SSLError, CertificateError) as e:
                 # Test whether the exception is due to the fact that this is an older version of
                 # Python that cannot validate certificates encrypted with SHA-2. If it is, then
```

Comment:
This comment talks about Python 2.5 and below. We might want to clean that up. I started that in #347.